package.json

```diff
@@ -31,12 +31,19 @@
   ],
   "devDependencies": {
     "ganache-cli": "^6.1.8",
+    "husky": "^1.1.4",
     "lerna": "^3.4.3",
     "prettier": "1.15.2",
+    "shasum-cli": "^0.0.1",
     "tslint": "^5.11.0",
     "tslint-config-airbnb": "^5.11.0",
     "tslint-config-prettier": "^1.15.0",
     "tslint-plugin-prettier": "^2.0.0",
     "typescript": "^3.1.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "cat packages/*/package.json | ./node_modules/.bin/sha256sum > .dependencies_checksum"
+    }
```

Comment:
Couldn't we just take this command and run it inside `circleci` before the cache is loaded and then we wouldn't need to add this to the repo?

Reply:
We could